exercises/twelve-days/description.md

```diff
@@ -1,6 +1,12 @@
-# Description
+# Instructions
 
-Output the lyrics to 'The Twelve Days of Christmas'.
+Your task in this exercise is to write code that returns the lyrics of the song: "The Twelve Days of Christmas."
+
+"The Twelve Days of Christmas" is a common English Christmas carol. Each subsequent verse of the song builds on the previous verse.
+
+The lyrics your code returns should _exactly_ match the full song text shown below.
+
+## The Twelve Days Of Christmas
```

Comment:
I'm wondering whether it might be clearer to have this named:

```suggestion
## Lyrics
```

We're already in the context of Twelve Days of Christmas

Reply:
Agreed!